app/assets/javascripts/editor.js

```diff
@@ -178,6 +178,7 @@ class Editor {
     this.previewElement[0].innerHTML = marked(this.textAreaValue);
     this.previewElement.toggle();
     commentFormBody.toggle();
+    commentFormBody.parent().find(".btn-toolbar").toggle();    // toggles comment toolbar
```

Comment:
Amazing work @KarishmaVanwari!! This is great!

All that's left is an updated screen recording of the UI changes. Do you think you could visit a note with many comments to check if everything works fine? You could try to edit a published comment and see if the toolbar hides when you click preview. Do the same for a reply to a comment and a new comment on its own.

Again, you've done really well.

Reply:
Thank you so much @TildaDares !! 
I've checked that the toolbar hides both on reply to a comment and publishing a comment on a page with multi comments existing, when the preview mode is on.
Do I have to record a video of the same?

Reply:
@KarishmaVanwari I've confirmed that too but it'd be better if you make a screen recording for documentation purposes. Thanks!

Reply:
@TildaDares  Okay, sure, I'll make one.